<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CardWindow</class>
 <widget class="QMainWindow" name="CardWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Card Description</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/img/resources/img/icon.png</normaloff>:/img/resources/img/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #454c5a;</string>
  </property>
  <property name="iconSize">
   <size>
    <width>128</width>
    <height>128</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="label_card_desc">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Torus Pro</family>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.5, y1:0.5, x2:0.95, y2:0.5, stop:0 rgba(107, 173, 238, 255), stop:1 rgba(69, 76, 90, 255));
color: #ffffff;
padding: 0 0 0 10px;</string>
      </property>
      <property name="text">
       <string>Card Description</string>
      </property>
      <property name="margin">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <property name="leftMargin">
        <number>15</number>
       </property>
       <property name="topMargin">
        <number>15</number>
       </property>
       <property name="rightMargin">
        <number>15</number>
       </property>
       <property name="bottomMargin">
        <number>15</number>
       </property>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: #f4f5f7;
border-radius: 5px;</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>12</number>
          </property>
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QLabel" name="label_title">
            <property name="font">
             <font>
              <family>Torus Pro</family>
              <pointsize>14</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #282c33;</string>
            </property>
            <property name="text">
             <string>Title</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_title">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Noto Sans</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
	background-color: #ebecf0;
	color: #282c33;
	border-radius: 5px;
	padding: 0 8px 0 8px
}

QLineEdit:focus {
	background-color: #ffffff;
	border-color: #6badee;
	border-width: 1.5px;
	border-style: solid;
	padding: 0 6px 0 6px
}

QTextEdit QMenu::item:selected {
	background-color: #6badee;
	color: #ffffff;
}
</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
            <property name="placeholderText">
             <string>Add a card title...</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QLabel" name="label_date">
                <property name="font">
                 <font>
                  <family>Torus Pro</family>
                  <pointsize>14</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: #282c33;</string>
                </property>
                <property name="text">
                 <string>Date</string>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCalendarWidget" name="calendarWidget">
                <property name="font">
                 <font>
                  <family>Noto Sans</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="focusPolicy">
                 <enum>Qt::TabFocus</enum>
                </property>
                <property name="styleSheet">
                 <string notr="true">QCalendarWidget QWidget {
	background-color: #6badee;
}

QCalendarWidget QAbstractItemView:!enabled {
	color: #cecece
}

QCalendarWidget QAbstractItemView:enabled {
	color: #000000
}

QCalendarWidget QToolButton {
	padding: 5px;
}

QCalendarWidget QMenu {
	background-color: #f5f5f5;
	border: none;
	padding: 2px;
	margin: 0px;
	font-size: 14px;
}

QCalendarWidget QMenu::item {
	padding: 5px;
}

QCalendarWidget QMenu::item:selected {
	background-color: #007bff;
	color: #ffffff;
}

QCalendarWidget QToolButton {
	icon-size: 18px;
}

QCalendarWidget QToolButton#qt_calendar_prevmonth {
	qproperty-icon: url(:/img/resources/img/left-arrow.png);
}

QCalendarWidget QToolButton#qt_calendar_nextmonth {
	qproperty-icon: url(:/img/resources/img/right-arrow.png);
}

QCalendarWidget QToolButton:pressed {
	background-color: #007bff;
}

QCalendarWidget QSpinBox {
	border: none;
	color: #000000;
}

QCalendarWidget QSpinBox::up-button,
QCalendarWidget QSpinBox::down-button {
	border-radius: 5px;
}

QCalendarWidget QSpinBox::up-arrow {
	image: url(:/img/resources/img/up-arrow.png);
	width: 10px;
	height: 10px
}

QCalendarWidget QSpinBox::down-arrow {
	image: url(:/img/resources/img/down-arrow.png);
	width: 10px;
	height: 10px
}

QCalendarWidget QSpinBox::up-arrow:pressed,
QCalendarWidget QSpinBox::down-arrow:pressed {
	width: 7px;
	height: 7px;
}
</string>
                </property>
                <property name="gridVisible">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="label_time">
                <property name="font">
                 <font>
                  <family>Torus Pro</family>
                  <pointsize>14</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: #282c33;</string>
                </property>
                <property name="text">
                 <string>Time</string>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTimeEdit" name="timeEdit">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Noto Sans</family>
                  <pointsize>16</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="styleSheet">
                 <string notr="true">QTimeEdit {
	background-color: #6badee;
	color: #ffffff;
	border-radius: 5px;
	padding: 0 5px 0 5px
}

QTimeEdit:focus {
	border-color: #000000;
	border-width: 1.5px;
	border-style: solid;
	padding: 0 3px 0 3px
}

QTimeEdit::up-button,
QTimeEdit::down-button {
	border-radius: 5px;
}

QTimeEdit::up-arrow {
	image: url(:/img/resources/img/up-arrow.png);
	width: 15px;
	height: 15px
}

QTimeEdit::down-arrow {
	image: url(:/img/resources/img/down-arrow.png);
	width: 15px;
	height: 15px
}

QTimeEdit::up-arrow:pressed,
QTimeEdit::down-arrow:pressed {
	width: 12px;
	height: 12px;
}
</string>
                </property>
                <property name="timeSpec">
                 <enum>Qt::LocalTime</enum>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>0</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>0</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_description">
            <property name="font">
             <font>
              <family>Torus Pro</family>
              <pointsize>14</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #282c33;</string>
            </property>
            <property name="text">
             <string>Description</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textEdit_description">
            <property name="font">
             <font>
              <family>Noto Sans</family>
              <pointsize>12</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QTextEdit {
	background-color: #ebecf0;
	color: #282c33;
	border-radius: 5px;
	padding: 4px 8px 4px 8px
}

QTextEdit:focus {
	background-color: #ffffff;
	border-color: #6badee;
	border-width: 1.5px;
	border-style: solid;
}

QScrollBar:vertical {
	width: 10px;
	margin: 0 0 0 0;
	background-color: #acb2bf
}

QTextEdit QMenu {
	background-color: #f5f5f5;
	border: none;
	padding: 5px;
	margin: 0px;
	font-size: 14px;
}

QTextEdit QMenu::item {
	padding: 5px;
}

QTextEdit QMenu::item:selected {
	background-color: {modify_hex_color(self.color)};
	color: #ffffff;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="autoFormatting">
             <set>QTextEdit::AutoAll</set>
            </property>
            <property name="tabChangesFocus">
             <bool>true</bool>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Torus Pro';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="tabStopDistance">
             <double>20.000000000000000</double>
            </property>
            <property name="acceptRichText">
             <bool>false</bool>
            </property>
            <property name="placeholderText">
             <string>Add a more detailed description...</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="btn_delete">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>140</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Torus Pro</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="focusPolicy">
               <enum>Qt::TabFocus</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
	background-color: #d63a3e;
	color: #ffffff;
	border-radius: 5px
}

QPushButton:hover {
	background-color: #9e2a2a;
}

QPushButton:focus {
	border-color: #000000;
	border-width: 1.5px;
	border-style: solid;
}
</string>
              </property>
              <property name="text">
               <string>Delete Card</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btn_cancel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Torus Pro</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="focusPolicy">
               <enum>Qt::TabFocus</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
	background-color: #acb2bf;
	color: #ffffff;
	border-radius: 5px
}

QPushButton:hover {
	background-color: #7e828c;
}

QPushButton:focus {
	border-color: #000000;
	border-width: 1.5px;
	border-style: solid;
}
</string>
              </property>
              <property name="text">
               <string>Cancel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_save">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Torus Pro</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="focusPolicy">
               <enum>Qt::TabFocus</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
	background-color: #6badee;
	color: #ffffff;
	border-radius: 5px
}

QPushButton:hover {
	background-color: #5487bb;
}

QPushButton:focus {
	border-color: #000000;
	border-width: 1.5px;
	border-style: solid;
}
</string>
              </property>
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_title</tabstop>
  <tabstop>calendarWidget</tabstop>
  <tabstop>timeEdit</tabstop>
  <tabstop>textEdit_description</tabstop>
  <tabstop>btn_delete</tabstop>
  <tabstop>btn_cancel</tabstop>
  <tabstop>btn_save</tabstop>
 </tabstops>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
